native-wheels-frontend/app/(tabs)/profile.tsx

```diff
@@ -1,90 +1,74 @@
 import Ionicons from '@expo/vector-icons/Ionicons';
-import { StyleSheet, Image, Platform } from 'react-native';
+import { StyleSheet, Image } from 'react-native';
+import { useState, useCallback } from 'react';
+import { useFocusEffect } from '@react-navigation/native';
+import { getBookings } from '@/axios/booking/api'; // Adjust the import path according to your project structure
 
-import { Collapsible } from '@/components/Collapsible';
-import { ExternalLink } from '@/components/ExternalLink';
 import ParallaxScrollView from '@/components/ParallaxScrollView';
 import { ThemedText } from '@/components/ThemedText';
 import { ThemedView } from '@/components/ThemedView';
+import { CarBanner } from '@/components/cars/CarBanner'; // Adjust the import path according to your project structure
+
+type Booking = {
+  _id: string;
+  carId: {
+    model: string;
+    make: string;
+    year: number;
+    price: number;
+    imageUrl: string;
+  };
+  fromBookingDate: string;
+  toBookingDate: string;
+};
```

Comment:
I would suggest moving this to a new file in `axios/booking/types.ts`. This way it follows the structure of the other api calls.

This type can then also be used as return type for the `getBookings()` in the `axios/booking/api.ts` which currently has no type.